jb/itch5/order_book.hpp

```diff
@@ -174,11 +220,8 @@ class order_book {
   }
 
 private:
-  typedef std::map<price4_t, int, std::greater<price4_t>> buys;
-  typedef std::map<price4_t, int, std::less<price4_t>> sells;
-
-  buys buy_;
-  sells sell_;
+  typename book_type::buys_t buy_;
```

Comment:
Ping.  I would feel better if you replied to the comment and said "Ack."  meaning "I am going to go with the current approach in this change, and consider the others for the change".